R/get_parameters.R

```diff
@@ -68,6 +75,12 @@ get_parameters <- function() {
         val <- read.csv(val)
       }
     }
+
+    # if val is still NULL, a default value exists for this parameter
+    if (is.null(val)) {
+      val <- params_config[[name]]$default
+    }
```

Comment:
Is this actually true? I am not getting why there has to be a `$default` if the val is `NULL`. Could you explain?

Reply:
I think the comment is clear, I was just missing the logic.